server/controllers/download-by-content-id.js

```diff
@@ -66,34 +66,40 @@ module.exports = exports = async (req, res, next) => {
 
 	prepareDownloadResponse(res, content);
 
-	if (dl.downloadAsArchive) {
-		dl.on('error', (err, httpStatus) => {
-			log.error('DOWNLOAD_ARCHIVE_ERROR', {
-				event: 'DOWNLOAD_ARCHIVE_ERROR',
-				error: err.stack || err
-			});
-
-			res.status(httpStatus || 500).end();
-		});
-
-		dl.on('end', () => {
-			log.debug(`DownloadArchiveEnd => ${content.id} in ${Date.now() - dl.START}ms`);
+	let articleOrArchive = dl.downloadAsArchive ? 'Archive' : 'Article'
```

Comment:
**Praise:** NICE solution! 🙌

Reply:
**Nitpick:** Please could we add a `;` to the end of the line?

Reply:
done